libs/portals/admin/petition/src/shared/queries/getAllEndorsementsLists.graphql

```diff
@@ -0,0 +1,21 @@
+query endorsementSystemFindEndorsementLists(
+  $input: PaginatedEndorsementListInput!
+) {
+  endorsementSystemFindEndorsementLists(input: $input) {
+    totalCount
+    pageInfo {
+      hasPreviousPage
+      hasNextPage
+      startCursor
+      endCursor
+    }
+    data {
+      id
+      title
+      description
+      closedDate
+      openedDate
+      adminLock
+    }
+  }
+}
```

Comment:
Recommend colocating graphql queries and mutations where they're used, i.e. in the screen folder with the loader.

That way we can minimise sharing and easily strip out unneeded fields. Can even join together graphql fragments from subcomponents.